plugins/woocommerce/client/legacy/js/frontend/a8c-address-autocomplete-service.js

```diff
@@ -0,0 +1,485 @@
+( function () {
+	const permanentlyDisabledServices = [];
+	const baseUrl =
+		'https://public-api.wordpress.com/wpcom/v2/woo/address-autocomplete';
+	const searchUrl = `${ baseUrl }/search`;
+	const selectUrl = `${ baseUrl }/select`;
+
+	const MAX_SERVICE_ERROR_RETRIES = 3;
+
+	/**
+	 * Debounce function from lodash, modified to return a promise.
```

Comment:
I needed a good debounce function, I'm not aware of a way to include npm modules in this scripting world, so I'm just copy pasting the whole function, with some modification, you can see a diff here https://www.diffchecker.com/TINeimGv/